src/commands/integrations/flows/test.ts

```diff
@@ -85,7 +85,14 @@ export default class TestFlowCommand extends PrismaticBaseCommand {
     "result-file": Flags.string({
       char: "r",
       description:
-        "Optional file to append tailed execution result data to. Results are saved as comma-separated values.",
+        "Optional file to append tailed execution result data to. Results are saved into JSON Lines.",
+    }),
+    jsonl: Flags.boolean({
+      description: "Optionally format the step and tail results output into JSON Lines.",
+    }),
+    succinct: Flags.boolean({
+      char: "s",
+      description: "Removes warnings and descriptive messaging from test output.",
```

Comment:
Should this be removed in favor of using the existing CLI-level `quiet` flag?

Reply:
Oh good point. Will update